src/features/collapsible-sidebar/__tests__/CollapsibleSidebarLogo.test.js

```diff
@@ -32,4 +32,20 @@ describe('components/core/collapsible-sidebar/CollapsibleSidebar', () => {
 
         expect(sidebar).toMatchSnapshot();
     });
+
+    test('render logo element with url thats passed', () => {
+        const sidebar = getWrapper({
+            url: '/some/url',
+        });
+
+        expect(sidebar).toMatchSnapshot();
```

Comment:
instead of snapshot, let's explicitly search for a `LinkBase` component with `/some/url` prop. Relying on snapshots can mask bugs later.